### ISSUE TITLE
Big push for 2019

### ISSUE THREAD

**Issue by cdsouthan** [OpenSourceMalaria/OSMSeries4Paper1#25]
Good to see progress on the draft but what's left to be done and who's gonna do it?   We now need a fork of the master list as a suppdat table for a figshare csv.   We should add just the potency figures that are on the structure pictures.  I am not going to take on the task of submitting 100's of our missing structures to PubChem (maybe we could consolidate these later).  For most of the inactives its not that valuable anyway.  What I started to do and could continue, is just get the front-runners with a 100 nM cut-off in as SIDs  (preferably with each having been resynthesized and re-tested).  I think I could manage the BioAssay entries for those as well.

**Comment by mattodd** [comment]
Great. I've three other papers I must submit in the short term before committing to this, but @edwintse may have some time to devote to the paper in the next few weeks. So bear with us before we can drive this again. Forking the list seems like something we ought to do late in the day, no, when we know which compounds are to be included and/or highlighted? What's your view on that timing?

**Comment by cdsouthan** [comment]
I'd suggest we are rather late in the day anyway ( three years between papers, but no one's fault).  Better to declare "data stop" and for authors to down tools, get the beers and pizza in and finish writing with what we have (more than enough, we should check with ACS Cen Sci about maximum lengh)  Its a natural pause now also since several folks have moved on (I had a pleasant catch-up beer with @david1597 in Boston recently)

**Comment by cdsouthan** [comment]
With that review out time for the team to get back into this perhaps

**Comment by danaklug** [comment]
Hi all, I am a new postdoc working with Mat at UCL and he's asked me to have a look at the draft. I've done a pretty serious overhaul of the Google doc ("V2" in the version history - "V1" is the old version). Major changes are highlighted below:

- I have laid out sections of the paper organized as best I can for a clear narrative, and the navigation pane should now reflect major sections and subsections clearly. 

- For the SAR tables, I have tried to capture the major points of what is highlighted in the wiki while being as concise as possible. Compounds that are not highlighted in the main text can go in the SI, or of course we can add them back in. 

- Text highlighted in yellow indicates a section yet to be written or outstanding questions/issues.

**Comment by MedChemProf** [comment]
@danaklug I finished updating the table that I started yesterday regarding the substituted amines and reversed amides/sulfonamide. What is you next list of items to be tackled? Are there other draft pages of the paper? Thanks.

**Comment by danaklug** [comment]
That's awesome, thanks! We would need to migrate it over to the [actual draft](https://docs.google.com/document/d/1vlOHNo9bxmx24KmNdJz5ZmjXyVP9uZoHJcnGj181NZE/edit) and then I think the SAR section will be fairly complete. @edwintse mentioned to me that some analogs were going to go in their own, smaller papers, so he's agreed to take a look and see if there's anything that needs to be taken out.

As far as actual writing, the introduction and the synthesis schemes are probably the biggest missing pieces right now, but anything highlighted in yellow still needs to be addressed. We'll also need to add results for the OHOH compound when those experiments are run.

I've structured the paper on the assumption that the OHOH compound will be the punch line and tried to build the narrative around that. Any feedback on the overall organization/structure of the paper would be really helpful.

**Comment by MedChemProf** [comment]
@danaklug Thank you and let me know if anything I added needs to be taken out. I did notice that my chemical images did not match the ones that were used in the other tables. Should I just grab one of the ChemDraw files from the GitHub repository to use as a template? Or is there another that you preferred to be used. I have schemes and experimental procedures already written up for all compounds added into the tables.

**Comment by cdsouthan** [comment]
@danaklug, Excellent to see this (a tad overdue...) progress towards "the"  paper.  As a minor author, I'll leave the meaty bits to those more worthy but I do have some suggestions on wrangling the data for surfacing into the paper, as Figshare supplementary tables and eventually into PubChem.  

Would it be possible to fork the master data sheet into a definitive sub-sheet of just those compounds to be included?  The multiple assay results can be trimmed back to just the averaged single values used in the SAR discussions but of course leave in the SMILES, InChI strings and InChIKeys.  You could possibly leave the manuscript section numbering (1-20+ etc) out since these will be clearly mapped to the code numbers in the text.   

I can then (recklessly) offer to do two things. Firstly, map all the InChIKeys into PubChem to see what is out there or not.  Secondly, I can attempt to submit say just the five front-runners (sub 100nM?) to PubChem BioAssay.  I don't have the time or resources to do this for all of Series 4 but once the SAR tables are in Figshare and the paper published we can see what might be doable, including PubChem taking the sheet as-is

So here's to bashing on :)

**Comment by danaklug** [comment]
@MedChemProf Will do. I used the ACS Document 1996 settings on ChemDraw... we'll have to rectify that eventually.

I think the biggest issue with the tables right now is that they're long and there are a lot of them. I'd like to go through and move some entries to the SI so that the paper is a bit more manageable. @cdsouthan Once this is finalized, I can start a sub-sheet with the columns you mentioned. Would it be helpful to also add columns indicating whether we have the requisite experimental/characterization? I've found that a helpful way to keep track of that sort of thing in the past.

If this sounds like a plan, I will do my best to get it all done by the end of the week and post updates here. Afterwards, we can start tackling the SI tables and putting the experimental together.

Thanks so much to you both for your input, it's really helpful!

**Comment by cdsouthan** [comment]
Fine, AWAK we passed the point of too much data for a concise publishable unit some time back (but no one's fault :). As said,  I defer to the esteemed senior medchemeratti (such as @MedChemProf  @mattodd et al ) to finalise content but It would seem OK to leave stuff out (e.g. SAR dead-ends) particularly as everything we don't mention is nonetheless open and Google-findable.  Along these lines can we not use the open  ELN urls to point across to synthetic details rather than replicate them back in and bloat the manuscript?  (but need to check if Journal ed is OK)  with this)

**Comment by mattodd** [comment]
Just surfacing v quickly from rural Portugal to say that the great work @danaklug has been doing will help us define the compounds that are going to appear in the final manuscript. The others can appear in SI or within OSM online infrastructure. I can see advantages in a spreadsheet of the compounds in the main paper, but would caution against creating this too early. Forks make me nervous. Anyone seriously interested in the series will always want to look at the full sheet. (End of surface).

**Comment by MedChemProf** [comment]
@danaklug @cdsouthan @mattodd All excellent suggestions. Due to the number of tables, the suggestion to narrow the focus of each to select compounds has a lot of merit to make the paper more readable. The complete table of compounds can be contained in the supplementary materials along with the experimental details and most of the synthetic schemes. I think the supplemental section should have both the experimental details written out (or at least for a representative molecule) along with the spectroscopic information. A link to the ELN can be included for each of the compounds. I have been keeping a spreadsheet with the final compounds synthesized that includes SMILES, IUPAC Name, ELN Link, MW, mp, 1H NMR, 13C NMR, and LCMS.

One thing that I am having a hard time wrapping my head around is how to most easily differentiate an inherited molecule (that we may not have full characterization) versus newer compounds the group synthesized? Perhaps we use an inherited molecule as a lead for each of the tables and then representative compounds the group made to flesh out the SAR?

**Comment by danaklug** [comment]
@MedChemProf I agree, it's a bit of a challenge to deal with. In many cases, inherited compounds are key SAR points and the follow-up analogs are not sufficient to tell the story. Is this something we could deal with by maybe flagging inherited compounds in table footnotes and addressing generally in the intro text?

For the SI, here's an example of a table that would go in as I'm thinking about it. Much simpler without including the structures like we have in the full paper, but the compounds are grouped by structural similarity, which is an advantage over just linking to the master spreadsheet. If this looks good, I can complete the rest of the tables and start a Google doc for the SI.
![image](https://user-images.githubusercontent.com/53487492/63452923-63ed7180-c43f-11e9-97c7-a63e47830c42.png)

**Comment by danaklug** [comment]
Created a new issue for the SI [here](https://github.com/OpenSourceMalaria/OSMSeries4Paper1/issues/28) - let me know what you think.

**Comment by MFernflower** [comment]
@danaklug Please kindly proof read the Evaluation against other pathogens sections as I have added a few points of note to it and since this would be my first paper I dont really know what to proof read for

**Comment by MFernflower** [comment]
@mattodd closing this since stuff has been broken off to other issues and now editing sessions are planned
